src/canmatrix/canmatrix.py

```diff
@@ -1843,7 +1861,12 @@ def __init__(self, arbitration_id):  # type: (ArbitrationId) -> None
         if arbitration_id.extended:
             self.source = arbitration_id.id & int('0xFF', 16)
```

Comment:
maybe if you're tochuing the code anyway, also refactor the two crazy `int('0xFF', 16)` to simply `0xFF`